src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Media/MediaPlayerState.cs

```diff
@@ -877,7 +877,7 @@ private void OpenMedia(Uri source)
                 {
                     // target is HTTPS. Then, elevate ONLY if we are NOT coming from HTTPS (=XDomain HTTPS app to HTTPS media disallowed)
                     Uri appDeploymentUri = SecurityHelper.ExtractUriForClickOnceDeployedApp();
-                    if (!SecurityHelper.AreStringTypesEqual(appDeploymentUri.Scheme, Uri.UriSchemeHttps))
+                    if (appDeploymentUri != null && !SecurityHelper.AreStringTypesEqual(appDeploymentUri.Scheme, Uri.UriSchemeHttps))
```

Comment:
@ojhad Is this still around after your security fixups?

Reply:
All this code is removed as part of the cleanup in this PR: https://github.com/dotnet/wpf/pull/969
So I don't think this should be an issue anymore.

Reply:
Looks like it! I'll close this if 969 gets merged.